newIDE/app/src/UI/Theme/NordTheme/index.js

```diff
@@ -94,7 +95,7 @@ const gdevelopTheme = {
   eventsSheetRootClassName: 'gd-events-sheet-nord-theme', // See EventsSheet.css
   tableRootClassName: 'gd-table-nord-theme', // See Table.css
   markdownRootClassName: 'gd-markdown-nord-theme', // See Markdown.css
-  gdevelopIconsCSSFilter: '',
+  gdevelopIconsCSSFilter: 'brightness(110%) hue-rotate(5deg) contrast(90%)',
```

Comment:
Since the icons have bluish colour to them, a slight hue-rotation gives them a purplish tint, this should blend in with the color palette of the them, but I'll be experimenting a bit more :)

![Screenshot from 2020-05-06 23-09-17](https://user-images.githubusercontent.com/39851078/81210745-f2b31600-8fef-11ea-9379-31372446ff49.png)

Reply:
I think we may want something that is a bit more desaturated. I have the feeling is hard to distinguish the purple from the background. I would almost go to black-and-white icons, with a shade that looks like the "Frost" colors:
![image](https://user-images.githubusercontent.com/1280130/81215228-49f9bc00-8fd9-11ea-8447-da27980213f0.png)

Not sure how doable it is, but for now they seem a bit hard to read on this background.

Reply:
@4ian I've used some hsl value transformations to bring in the frost colour into the icons :D 

![Screenshot from 2020-05-07 01-47-15](https://user-images.githubusercontent.com/39851078/81224218-0157f800-9005-11ea-96ed-ece035ce4152.png)

The frost color in icons is quite accurate at some places, with the exact same hsl value (rgb varies slightly) of the nord frost. The obvious problem is that the background of these icons is frost, whereas in the picture you posted above, we can see that the background is dark. 

The icons don't really have that sort of black and white, meaning if I'd use grayscale, either they'll become completely white or completely black :/ 

Anyway, I'd experiment a bit more to see what more can be done, hopefully finding a way to bring some contrast between background and foreground of the icons :) Thank you for your suggestions and inputs :D

Reply:
> The icons don't really have that sort of black and white, meaning if I'd use grayscale, either they'll become completely white or completely black :/

Yep, we won't find a perfect solution - apart from switching to SVG icons. But that's other of the scope of this PR.
Give a bit of time to see what's the best you can find - it's ok to diverge a bit from colors as long as it's now too shocking and icon details are visible enough :)